bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/ValueCreator.java

```diff
@@ -90,6 +90,10 @@ public static ValueCreator getValueCreator(String key) {
         return runtimeValueCreators.get(key);
     }
 
+    public Object call(Strand strand, String funcName, Object... args) throws BError {
+        throw new BallerinaException();
```

Comment:
Shall we retuen  something like new ErrorValue(StringUTils.fromString(No such method + fucntionnAme);